### PR TITLE
WINC-1199: Fix VM creation automation in new vCenter

### PR DESCRIPTION
**Pull request by saifshaikh48**
- Updates packer config to work on new DEV/QE environment
- Adds scripts and documentation to copy over VMs across vCenter environments, to be used only as a last resort to move existing VMs (created via packer or otherwise) to CI environments.

### CODE REVIEW

File: docs/vsphere_ci/windows-server-2022.pkr.hcl [hunk 1]
```diff
@@ -78,8 +83,7 @@ source "vsphere-iso" "windows-server-2022" {
   insecure_connection  = "true"
   iso_paths            = ["${var.os-iso-path}", "${var.vmtools-iso-path}"]
   network_adapters {
-    network      = "dev-segment"
-    network_card = "vmxnet3"
```
Comment (aravindhp) [inline docs/vsphere_ci/windows-server-2022.pkr.hcl:82]: Is there a reason to drop this? You might default to a non-paravirtualized network device which will result in reduced network performance which can be important when pulling images.

Comment (saifshaikh48) [reply docs/vsphere_ci/windows-server-2022.pkr.hcl:82]: Nice catch, I had dropped it when I was using a different (incorrect) network with a different network adapter and forgot to add it back in

**Comment by jrvaldes** [comment]
@mansikulkarni96  @sebsoto   consider overriding the ci jobs in this PR given the changes are docs-related.

**Review by sebsoto — APPROVED** [review]

File: docs/vsphere_ci/vcenter_copy_template/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Install `govc` on the host where you will be building image. Download, unzip, and move the utility within your `$PATH`.
+e.g.
+
+```
+curl -L -o - "https://github.com/vmware/govmomi/releases/latest/download/govc_$(uname -s)_$(uname -m).tar.gz" | tar -C /usr/local/bin -xvzf - govc
+```
+
+## Prerequisite files
+
+Please ensure the following file are present in the location where you are running the cloning scripts from:
```
Comment (wgahnagl) [inline docs/vsphere_ci/vcenter_copy_template/README.md:21]: nit: 
following file -> 
following files

File: docs/vsphere_ci/vcenter_copy_template/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Fill out the config variables with your credentials for each environment, 1 DEV/QE and 3 CI.
+
+## Scripts
+
+There are two scripts to run, each of which requires passing in a source file to populate govc config variables.
+
+    - getem.sh
+    - putem.sh
+
+You can run these from a "jump" VM in vSphere or from your local machine, but the latter will be slower in the transfer.
```
Comment (wgahnagl) [inline docs/vsphere_ci/vcenter_copy_template/README.md:38]: this seems a little unclear. Could there be more info here about what a jump VM means?

File: docs/vsphere_ci/vcenter_copy_template/sourcevcs8e [hunk 4]
```diff
@@ -0,0 +1,6 @@
+export GOVC_URL=https://vcs8e-vc.ocp2.dev.cluster.com
+export GOVC_USERNAME=xxxxxxx@vsphere.local
+export GOVC_PASSWORD='xxxxxxx'
+export GOVC_INSECURE=true
+export GOVC_DATACENTER=IBMCloud
+export GOVC_RESOURCE_POOL='QE Resource Pool'
```
Comment (wgahnagl) [inline docs/vsphere_ci/vcenter_copy_template/sourcevcs8e:6]: these templates seem very similar to each other
Is there a way to combine them?

**Comment by jrvaldes** [comment]
/hold

for @wgahnagl  comments. 

@saifshaikh48  PTAL

File: docs/vsphere_ci/vcenter_copy_template/sourcevcs8e (hunk 4 above)
Comment (saifshaikh48) [reply docs/vsphere_ci/vcenter_copy_template/sourcevcs8e:6]: not really, they are example configs for different vcenter envs which each may have different creds

**Comment by jrvaldes** [comment]
LGTM

@wgahnagl  will defer the ` / lgtm` to you

**Comment by sebsoto** [comment]
/override ci/prow/aws-e2e-operator ci/prow/aws-e2e-upgrade ci/prow/azure-e2e-operator ci/prow/azure-e2e-upgrade ci/prow/gcp-e2e-operator ci/prow/nutanix-e2e-operator ci/prow/platform-none-vsphere-e2e-operator ci/prow/vsphere-proxy-e2e-operator ci/prow/vsphere-e2e-operator ci/prow/wicd-unit-vsphere

**Comment by mansikulkarni96** [comment]
/override ci/prow/aws-e2e-operator ci/prow/aws-e2e-upgrade ci/prow/azure-e2e-operator ci/prow/azure-e2e-upgrade ci/prow/gcp-e2e-operator ci/prow/nutanix-e2e-operator ci/prow/platform-none-vsphere-e2e-operator ci/prow/vsphere-proxy-e2e-operator ci/prow/vsphere-e2e-operator ci/prow/wicd-unit-vsphere

**Pull request merged**